src/app/admin/requests/request-details/request-details.component.html

```diff
@@ -1,22 +1,295 @@
-<div class="mt-4">
-  <a href="#" appGoBack>&lt;- Înapoi</a>
+<div class="modal_header_line" fxLayout="row" fxLayoutAlign="center center">
+  <a>Crează Cerere Nouă</a>
```

Comment:
Why page for showing request details, was refactored into page to create request?

Reply:
The component was totally rewrote to cover issue #145. According to the Figma,  New Component and Modify existing one functionality have the same view component. So it take sense to have one component for both cases (mean request-details component), and, inside *.ts cover difference between logic of create new request (issue145) and modify existent request (issue not-exist).

Reply:
Start to modify according phone discussion